software/aliasvault.yml

```
@@ -0,0 +1,15 @@
+name: "AliasVault"
+website_url: "https://www.aliasvault.net"
+source_code_url: "https://gitlab.com/lanedirt/AliasVault"
+description: "A fully self-hostable, end-to-end encrypted password manager with a built-in alias generator and email server."
+licenses:
+  - MIT
+platforms:
+  - C#
+  - .NET
+  - Javascript
+  - Docker
```

> https://github.com/lanedirt/AliasVault?tab=readme-ov-file#install-using-install-script

The only platform mentioned here is docker, are the other ones needed to install or are they installed with the script?
```suggestion
platforms:
  - Docker
```

---

You're correct, only Docker is needed. All other dependencies are part of the Docker images themselves. I didn't correctly interpret the meaning of the platform property.